packages/oauth/oauth-scopes/src/scopes/include-scope.ts

```diff
@@ -0,0 +1,147 @@
+import { AtprotoAudience, isAtprotoAudience } from '@atproto/did'
```

Comment:
It would be nice to get some tests on this, esp. for limits on `lxm` and `collection`.